lib/controller/BibliotheekController.class.php

```diff
@@ -43,8 +43,10 @@ public function __construct($query) {
 
 	public function performAction(array $args = array()) {
 		parent::performAction($args);
-		$this->view = new CsrLayoutPage($this->view);
-		$this->view->addCompressedResources('bibliotheek');
+		if ($this->action != "autocomplete") {
+            $this->view = new CsrLayoutPage($this->view);
```

Comment:
Wut, vaag. Ik dacht dat ik editorconfig in PhpStorm aan had staan. Volgens mij is dit een geval waar de oude regels code nog wel met spaties indented waren. Ik ga die ook wel aanpassen dan.